reports/2022.html

```diff
@@ -199,51 +203,157 @@ <h3>Links</h3>
           <dt>Previous WCCG Report(s)</dt>
           <dd><a href="https://w3c.github.io/webcomponents-cg/index.html#form-associated-custom-elements">2021</a></dd>
           <dt>GitHub issues:</dt>
-          <dd>---</dd>
+          <dd><a href="https://github.com/whatwg/html/issues">https://github.com/whatwg/html/issues</a></dd>
           <dt>Browser positions:</dt>
-          <dd>---</dd>
+          <dd>
+            <ul>
+              <li>
```

Comment:
Is there a Mozilla stance link we could use? And should we link to chromestatus.com as well? Even when “shipped”.